.github/workflows/tests.yml

```diff
@@ -65,6 +65,8 @@ jobs:
           path: /usr/local/faasm/object
           key: ${{ runner.os }}-cache
       # --- Code build ---
+      - name: "Rerun cmake config"
```

Comment:
Let's actually remove this, as you say, it already gets run :thinking:

Reply:
Already done it locally, forgot to push :man_facepalming: